src/controller/cve.controller/index.js

```diff
@@ -22,10 +22,20 @@ router.get('/cve/:id',
     description: 'The requested CVE Record',
     content: {
       "application/json": {
-        schema: { $ref: '/schemas/cve/get-cve-record-response.json' }
+        schema: {
+            oneOf: [
+                { $ref: '/schemas/cve/get-cve-record-response.json' },
+                {$ref: '/schemas/cve/create-cve-record-rejection-response.json'}
+            ]
+        },
+        examples: {
+            Published: {$ref: '#/components/examples/publishedRecord'},
```

Comment:
This is cool. It would be more clear to say Published Record and Rejected Record for the select list